execution-engine/Cargo.toml

```diff
@@ -32,9 +32,9 @@ platform-services = { path = "../platform-services" }
 policy-utils = { path = "../policy-utils" }
 serde = { git = "https://github.com/veracruz-project/serde.git", features = ["derive"], branch = "veracruz" }
 typetag = { git = "https://github.com/veracruz-project/typetag.git", branch = "veracruz" }
-wasi-types = { git = "https://github.com/veracruz-project/wasi-types.git", branch = "veracruz" }
+wasi-types = { git = "https://github.com/egrimley-arm/wasi-types.git", branch = "veracruz" }
```

Comment:
Can you put these under the `veracruz-project` rather than your personal Github account?

Reply:
I'm hoping https://github.com/veracruz-project/wasi-types/pull/6 can be merged first.

Reply:
Ah OK, that makes sense.  I will take a look at that, now.